nucypher/network/teachers.py

```diff
@@ -17,6 +17,5 @@
 
 # Hardcoded bootstrapping teacher nodes keyed by network domain
 TEACHER_NODES = {
-    'cassandra': ('https://cassandra.nucypher.network:9151', ),
-    'gemini': ('https://gemini.nucypher.network:9151', )
```

Comment:
Curious what others think - no reason to keep these, right?

Reply:
Well, there are still hosts at those addresses... Not sure why!

Reply:
:grimacing: